des/car.py

```diff
@@ -1,14 +1,26 @@
+import numpy as np
 class Car:
 
     def __init__(self, id : int, type="hourly"):
         self.id = id
         self.type = type.lower()
 
+    def name(self):
```

Comment:
Nit: name might be unintuitive since you're retrieving the id attribute instead

Reply:
will change to `get_id`